src/main/webapp/app/view/create-project/project-create.component.scss

```diff
@@ -0,0 +1,23 @@
+#project-create-component {
+  @import '~bootstrap/scss/bootstrap';
+
+  .ngx-autocomplete-input {
+    @extend .form-control;
+  }
+
+  .ngx-autocomplete-container {
+    border-radius: $border-radius;
+    background-color: $body-bg;
+
+    .ngx-autocomplete-option {
+      border-radius: 0;
+
+      @extend .form-control;
+
+      &-active {
+        color: $dropdown-link-hover-color;
+        background-color: $dropdown-link-hover-bg;
+      }
+    }
+  }
+}
```

Comment:
Wollen wir das nicht lieber in einem allgemeinen SCSS einfügen? Ansonsten haben wir redudanten Code in den SCSS Files

Reply:
jop, finde ich ok 👍🏼